client/src/components/previews/EventPreview.js

```diff
@@ -0,0 +1,129 @@
+import API from "api"
+import { BreadcrumbTrail } from "components/BreadcrumbTrail"
+import DictionaryField from "components/DictionaryField"
+import { PreviewField } from "components/FieldHelper"
+import LinkTo from "components/LinkTo"
+import { Event } from "models"
+import moment from "moment"
+import PropTypes from "prop-types"
+import React from "react"
+import { ListGroup, ListGroupItem } from "react-bootstrap"
+import Settings from "settings"
+
+const EventPreview = ({ className, uuid }) => {
+  const { data, error } = API.useApiQuery(Event.getEventQuery, {
+    uuid
+  })
+
+  if (!data) {
+    if (error) {
+      return <p>Could not load the preview</p>
+    }
+    return null
+  }
+
+  const event = new Event(data.event)
+  const eventTitle = event.name || `#${event.uuid}`
+  return (
+    <div className={`report-preview preview-content-scroll ${className || ""}`}>
+      <h4 className="ellipsized-text">Event {eventTitle}</h4>
+      <div className="preview-section">
```

Comment:
Where are the `status` and `description` fields?

Reply:
It is a preview, I do not think we want to have description there in my opinion.